lib/util/tests/ert_util_strcat_test.cpp

```diff
@@ -28,8 +28,10 @@ void test_strcat(char * s1 , const char *s2 , const char * expected) {
   char * cat = util_strcat_realloc(s1 , s2 );
   if (test_check_string_equal( cat , expected ))
     free( cat );
-  else
-    test_error_exit("util_strcat_realloc(%s,%s) Got:%s  expected:%s \n",s1,s2,cat , expected);
+  else {
+    fprintf(stderr, "util_strcat_realloc(%s,%s) Got:%s  expected:%s \n",s1,s2,cat , expected);
+    exit(1);
```

Comment:
I am not sure I fully understand this change

Reply:
Hehe - that I can understand - the change was not even complete. The point was that the `expected` pointer leaked because the `test_error_exit( )` function terminated with a valid pointer, I guess the sanitizer stopped complaining when the `exit(1)` was "nearby"(?) - anyway I have now also added the missing `free( expected )`.